apps/web/src/routes/_content/resources/builders/builder.$builderName-$builderId.tsx

```diff
@@ -78,6 +96,14 @@ function Builder() {
       )
     : [];
 
+  useEffect(() => {
+    if (builder && params.builderName !== formatNameForURL(builder.name)) {
+      navigate({
+        to: `/builder/${formatNameForURL(builder.name)}-${builder.id}`,
```

Comment:
doesn't work (same for every other resources) -> "/resources" is missing in the url